esmvalcore/cmor/_fixes/cmip6/gfdl_cm4.py

```diff
@@ -0,0 +1,66 @@
+"""Fixes for GFDL-CM4 model."""
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+
+class Tas(Fix):
+    """Fixes for tas."""
+
+    def fix_metadata(self, cubes):
+        """
+        Add height (2m) coordinate.
+
+        Parameters
+        ----------
+        cube : iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        cube = self.get_cube_from_list(cubes)
+        add_scalar_height_coord(cube, 10.0)
```

Comment:
Sorry I missed this before, but of course for `tas` there should be no 10.0 since indeed it is at 2.0 m

Reply:
It is actually my fault, sorry. Busy doing other stuff, I am only performing the changes you requested, but I could have found it myself...